assets/styles/app.scss

```diff
@@ -1,2 +1,9 @@
-//start coding by removing this file and adding yours ;)
+$primary: (#0D9DDA);
+$secondary: (#002E5F);
 @import "~bootstrap/scss/bootstrap";
+@import url("https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.5/font/bootstrap-icons.css");
```

Comment:
No. You have correctly installed  bootstrap-icons with composer and yarn. So you this dependancy in your files, so you import it from a CDN. As bootstrap just above find a way to not use the CDN